### DISCUSSION TITLE
Task Bar icons now shows file name and application name

### DISCUSSION THREAD

**Discussion by MartinHugh** [valinet/ExplorerPatcher#2602] [General]
Is there a way to configure the task bar so that only icons are displayed. 
Having application name / filename is unnecessary and moves the position of the icons around

Thanks

**Comment by pyrates999** [comment]
here's a guide on customizing the windows 11 taskbar: https://www.anoopcnair.com/customize-windows-11-taskbar-best-method-option/

in the EP properties application, check the options in taskbar.

**Comment by MartinHugh** [comment]
Thank you for the reply. I am not sure you understood the question.
I am unable to find any configuration to prevent the stated problem.
Could you be more specific

Thank you

**Comment by tallman69** [comment]
How you accomplish that depends on which taskbar you are using - Windows 11 or windows 10.

If you are using the Windows 11 taskbar, right click on the taskbar and choose `Taskbar settings`, Expand `Taskbar behavior`, and set `Combine taskbar buttons and hide labels` to `Always`.  Then open the ExplorerPatcher Properties app and click on `Restart File Explorer` in the lower left corner.

If you are using the Windows 10 taskbar, open the ExplorePatcher Properties app, in the `Taskbar` tab, set `Combine taskbar icons on primary taskbar` to `Always Combine`.  If you have a secondary taskbar, do the same for it.

**Comment by MartinHugh** [comment]
Thank you that now works fine.

**Comment by pyrates999** [comment]
Please close this or mark the answer given as the answer.
